### PR TITLE
Test case for Issue #26

### PR DESCRIPTION
**Pull request by George-NG**
This is a test case for issue #26 - proves that this issue is no longer present.

### CODE REVIEW

File: t/issue26.t [hunk 1]
```diff
@@ -0,0 +1,20 @@
+use strict;
+use Test::More;
+use HTTP::Cookies;
+use HTTP::Response;
+use HTTP::Request;
+
+my $cookie_jar = new HTTP::Cookies;
```
Comment (skaji) [inline t/issue26.t]: Please don't use indirect method calls.

File: t/issue26.t [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+my $request = HTTP::Request->new(GET => 'http://www.en.com/');
+
+my $response = HTTP::Response->parse                           
+        ("HTTP/1.1 302 Moved\r\nSet-Cookie: expires=10101\r\n\r\n");
```
Comment (skaji) [inline t/issue26.t]: I don't think `\r\n` is portable.

**Comment by George-NG** [comment]
Apologies for my silly copy/paste mistake. I just re-used the code which was at the opening post of the issue. I hope you meant \r\n is the windows way for doing line endings and not some byte sequence.

**Comment by skaji** [comment]
Sorry, I mean we should use `"\015\012"` instead of `"\r\n"`.
cf https://github.com/libwww-perl/HTTP-Message/blob/master/lib/HTTP/Message.pm#L9

**Comment by George-NG** [comment]
After fixing the test which I added with the suggested value, the test was failing. Turns out we are actually "join"ing "\n" in the as_string() method. So after modifying that I had to fix the other test as well, but now both tests are passing.

It worked fine in my local tests, but I haven't got real use cases to test against.

**Comment by George-NG** [comment]
A simple test that became epic quest ;-)

Reverted the non-test files I've modified, I hope this time it is to your liking.

Thanks!

**Comment by skaji** [comment]
LGTM

**Comment by oalders** [comment]
@George-NG thanks so much for this.  Would you be able to squash this into a single commit and rebase?

**Comment by George-NG** [comment]
@oalders Is the last change OK?

**Comment by oalders** [comment]
@George-NG yes, thank you and thanks for following up with me on this.  I didn't see that you had pushed the commit.

**Pull request merged**